js/pages/ssn.js

```diff
@@ -1,4 +1,4 @@
-import { storeSocial, storeResponse, hideAnimation, translateHTML } from '../shared.js';
+import { storeSocial, storeResponse, hideAnimation, translateHTML, getSelectedLanguage } from '../shared.js';
```

Comment:
getSelectedLanguage is imported but not used

Reply:
thanks for the catch

Reply:
resolved